EveripediaNetwork/ep-ui
# Adding Lens Profile Feature

# [lens integration on profile#1205](https://github.com/EveripediaNetwork/issues/issues/1205)

Hi there!

I'm pleased to submit this PR for adding a new feature to our app. This PR adds the ability to add a lens profile to the app's settings, and also allows it to be displayed on the user's profile. With this new feature, users will be able to add information about the lenses they use, which will be displayed alongside their other profile information.

## How should this be tested?

1. Go to settings
2. and lens profile link
3. check you profile for the lens link

## Notes or observations
I used @kesar lens profile for testing

<img width="680" alt="image" src="https://user-images.githubusercontent.com/67792966/233353378-6419da2e-e28b-492f-b464-d941d05353c9.png">

<img width="692" alt="image" src="https://user-images.githubusercontent.com/67792966/233354348-c2beda68-c6fc-438b-ac38-0d4e0c8f79ae.png">

## Linked issues

fixes : https://github.com/EveripediaNetwork/issues/issues/1205
